### ISSUE TITLE
Cannot make div scrollable (mobile?)

### ISSUE THREAD

**Issue by piciuok** [react-component/drawer#134]
Hey!

I'm supprised that i cannot scroll my div. I looked deeper and i found, that method removeMoveHandler in DrawerChild prevent me from scrolling. Have you any idea how to solve that?

If i return; in that method immediately, all works fine. 

steps:
1. create div in drawer
2. add to div overflow:auto and max-width: 100%
3. create inside div another div, give him min-width: 1000px and some fake content
4. switch to mobile view, set 400px viewport
5. try to scroll by swipe that div

I use that for scrolling tables in articles, that have many columns and not fit to small screen

**Comment by sowka1995** [comment]
I have the same issue. If you set mask property to false than your drawer content will be scrollable but its not a solution, because we normally want to have a mask overlay - mainly on desktop

**Comment by anhtran** [comment]
I have to modify code to force `passive: true` in https://github.com/react-component/drawer/blob/master/src/DrawerChild.tsx#L313 to make the app works. Hope this issue will be resolved soon.

**Comment by jljsj33** [comment]
Android??? What browser???/

**Comment by anhtran** [comment]
@jljsj33 Chrome on iOS and Android.

**Comment by jljsj33** [comment]
iOS chrome no problem.

 
![Kapture 2020-06-04 at 19 24 02](https://user-images.githubusercontent.com/6802825/83751167-0017fa80-a699-11ea-9e63-f85ef4debf91.gif)

**Comment by jljsj33** [comment]
need reproduce

**Comment by anhtran** [comment]
@jljsj33 sorry, it's about horizontal scroll.

**Comment by jljsj33** [comment]
Problem found, https://github.com/react-component/drawer/blob/master/src/utils.ts#L107-L108

**Comment by jljsj33** [comment]
fix in v4.0.1

**Comment by luigi-veo** [comment]
> fix in v4.0.1

It is still not working for me, neither on chrome, safari or firefox. The scroll is locked.
I'm using version 4.3.1

It works fine with placement="left" or "right"
It does not work with "top" or "bottom"

```javascript
<Drawer
      open={isOpen}
      onClose={closeDrawer}
      level={null}
      placement="top"
      handler={false}
      wrapperClassName="instant-search__drawer"
   >{ children }</Drawer>
```
